helm-deploy/action.yml

```
@@ -69,15 +75,45 @@ runs:
         yq -i '.appVersion = "${{ github.sha }}" | .appVersion style="double"' \
           ${{ inputs.helm-package }}/Chart.yaml
 
+    - name: Collect Grafana dashboards and alerts
+      id: grafana-files
+      shell: bash
+      working-directory: ${{ inputs.working-directory }}
+      run: |
+        SET_FILE_ARGS=""
+        
+        if [[ -d "${{ inputs.grafana-dashboards-folder }}" ]] && [[ "$(ls -A "${{ inputs.grafana-dashboards-folder }}")" ]]; then
+          i=0
+          for f in "${{ inputs.grafana-dashboards-folder }}"/*; do
+            if [[ -f "$f" ]]; then
+              SET_FILE_ARGS+=" --set-file grafana.dashboards[$i]=$f"
+              ((i++))
+            fi
+          done
+        fi
+        
+        if [[ -d "${{ inputs.grafana-alerts-folder }}" ]] && [[ "$(ls -A "${{ inputs.grafana-alerts-folder }}")" ]]; then
+          i=0
+          for f in "${{ inputs.grafana-alerts-folder }}"/*; do
+            if [[ -f "$f" ]]; then
+              SET_FILE_ARGS+=" --set-file grafana.alerts[$i]=$f"
+              ((i++))
+            fi
+          done
+        fi
+        
+        echo "set-file-args=$SET_FILE_ARGS" >> $GITHUB_OUTPUT
+
     - name: Deploy Helm chart
       shell: bash
       working-directory: ${{ inputs.working-directory }}
       env:
         HELM_DRIVER: configmaps
       run: |
         helm upgrade --wait --install ${{ inputs.release-name }} ./${{ inputs.helm-package }} \
-          --namespace ${{ inputs.namespace }} \
           --history-max 10 \
-          --values ${{ inputs.values-file }} \
           --atomic --timeout ${{ inputs.timeout }} \
           --set image.tag=${{ inputs.image-tag }}
```

You forgot a backslash here
```suggestion
          --set image.tag=${{ inputs.image-tag }} \
```